### PR TITLE
Add handler E2E tests for static addresses and routes

### PR DESCRIPTION
**Pull request by yboaron**
Signed-off-by: Yossi Boaron yboaron@redhat.com

**Is this a BUG FIX or a FEATURE ?**:

> 
> /kind enhancement

**What this PR does / why we need it**:
This PR adds E2E static (both V4 and V6) addresses and routes tests for handler

### CODE REVIEW

**Review by qinqon — CHANGES_REQUESTED** [review]
Can you check if those state snippets are not already at https://github.com/nmstate/kubernetes-nmstate/blob/main/test/e2e/handler/states.go ? 

Also don't export the function.

**Review by qinqon — CHANGES_REQUESTED** [review]

File: test/e2e/handler/static_addr_and_route_test.go [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+    type: ethernet
+    state: up
+    ipv4:
+      address:
+      - ip: %s
+        prefix-length: %s
+      state: absent
+      enabled: false
+`, firstSecondaryNic, ipAddress, prefixLen))
+}
```
[the remark this answers is not in the document]
Comment (yboaron) [reply test/e2e/handler/static_addr_and_route_test.go:24]: Done

**Pull request merged**